deeppavlov/configs/ranking/paraphrase_ident_paraphraser.json

```diff
@@ -79,9 +79,9 @@
     "pytest_max_batches": 2,
     "train_metrics": ["f1", "acc", "log_loss"],
     "metrics": ["f1", "acc", "log_loss"],
-    "validation_patience": 10,
-    "val_every_n_epochs": 5,
-    "log_every_n_batches": 12
+    "validation_patience": 50,
```

Comment:
Are you sure about validation_patience of 50? That's a lot

Reply:
fixed.